groupnews-service/src/main/java/com/epam/internship/UserService.java

```diff
@@ -4,10 +4,14 @@
 
 import com.epam.internship.dto.User;
 
+import javassist.NotFoundException;
+
 public interface UserService {
 
 	User createUser(String name, String email);
 
 	List<User> getAllUsers();
 
+	User getUserById(Long id) throws NotFoundException;
```

Comment:
1. Please do not use javassist exceptions. For such purpuse, create a custom runtime exception.
2. It is better to use an unchecked exception in this case.